app/jobs/resolution_proofing_job.rb

```diff
@@ -14,7 +14,7 @@ class ResolutionProofingJob < ApplicationJob
 
   def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
               dob_year_only:, user_id: nil, threatmetrix_session_id: nil,
-              uuid_prefix: nil, request_ip: nil)
```

Comment:
this is set here https://github.com/18F/identity-idp/blob/85a46ff620b38bc6870d5641f64530128c60ad73/app/services/idv/steps/verify_base_step.rb#L153 and lives in `applicant_pii` already.

Additionally, it was never set in `idv/agents.rb` so it is safe to remove this argument here